BaseFilter/sections/specific.txt

```diff
@@ -16804,8 +16804,8 @@ outlook.live.com#?##app > div[class] > div[class] > div[class] > div[class]:not(
 outlook.live.com#?#.customScrollBar[data-is-scrollable] > div[class] > div[class]:not([data-convid]):has(> div:not([id]):not([class]) > div[class] > div[class] > div[class] > div[class] > div[class] > .fbAdLink)
 !
 ! check-host.net
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋|\﹢|х|Х|Χ)$/):upward(2)
-/^https:\/\/check-host\.net\/images\/[a-z]?[a-z]?[a-z]?[0-9]?[0-9]?[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯|✗|\+|\＋|\﹢|х|Х|Χ|Ẋ)$/):upward(2)
+check-host.net#?#img[src*=".png?"][alt^="DDOS-Guard"]
```

Comment:
Not needed as it is already blocked by the rule above.
I will add a new regex rule.